wolfssl/wolfcrypt/wc_port.h

```diff
@@ -158,8 +158,8 @@
 #else /* ! WOLFSSL_LINUXKM */
 
     #ifdef BUILDING_WOLFSSL
-        #define SAVE_VECTOR_REGISTERS() ({})
-        #define RESTORE_VECTOR_REGISTERS() ({})
+        #define SAVE_VECTOR_REGISTERS() do{}while(0)
+        #define RESTORE_VECTOR_REGISTERS() do{}while(0)
```

Comment:
Any reason to not surround this in parenthesis?  Better safe than sorry when dealing with the preprocessor.

Reply:
Not sure I understand the proposal
  do.. while(0) is an often used idiom, and has some advantages compared to macros written differently.

I do not think it needs more safety , as it is mostly safe. That ({}) added by  some Linux contribution in contrast was not even C. 

In fact, you can just make this macro empty, but then it SAVE_VECTOR_REGISTERS(); will have an extraneous semicolon.

Reply:
Thanks for explaining further, I agree it's fine.